admin/howto/manage-users.md

```diff
@@ -14,12 +14,7 @@ Users can prove who they are by logging in via an *authentication provider*. Cur
 
 2. [*GitHub*](https://github.com/). Extremely popular community of people creating, publishing and collaborating on code. Accounts are free, and many people already have them especially since the target community for most hubs are people who also write some kind of code. We can setup GitHub authentication so you can either manage a list of specific GitHub handles in the [JupyterHub admin panel](admin/management/admin-panel), or so that members of a specific GitHub organisation or team are automatically authorised to use the hub.
 
-3. Username / Password via [auth0](https://auth0.com/docs/connections/database).
-   A traditional username / password interface where users can sign up. There are currently [limited
-   options](https://github.com/2i2c-org/infrastructure/issues/421) for limiting who
-   can sign up, so this should be only used in limited circumstances.
-
-4. `<a different provider>`. We may be able to support other authentication providers, depending on your specific needs and the provider's complexity. Please reach out to us if none of these 3 work for your use-case.
+3. `<a different provider>`. We may be able to support other authentication providers, depending on your specific needs and the provider's complexity. Please reach out to us if none of these 3 work for your use-case.
```

Comment:
I replaced `these 3` with `those above` - knowing its hard to keep fixed numbers up to date anyhow!

```suggestion
3. `<a different provider>`. We may be able to support other authentication providers, depending on your specific needs and the provider's complexity. Please reach out to us if none of those above work for your use-case.
```

This list is outdated not mentioning that any kind of provided via CILogon can be supported, but I see this as an incremental improvement so it gets a approval!